src/utils.js

```diff
@@ -453,29 +453,19 @@ exports.insertElement = function(elm, doc, target) {
   } catch (e) {}
 };
 
-exports.insertPixel = function (url) {
+exports.triggerPixel = function (url) {
   const img = new Image();
-  img.id = _getUniqueIdentifierStr();
   img.src = url;
```

Comment:
I think it's fine being OK for this use case, I'm thinking about when this is used to count an impression we need to make sure it works. I'm ok as long as this was tested in all the browsers we care about.